pkg/scheduler/watchers/membercluster/controller_integration_test.go

```diff
@@ -444,6 +444,63 @@ var _ = Describe("scheduler member cluster source controller", Serial, Ordered,
 		})
 	})
 
+	Context("ready cluster has taint added, updated & removed", func() {
```

Comment:
we need a case that the new taint added won't kick the existing placed resources out of the cluster

Reply:
This test covers the case described https://github.com/Azure/fleet/blob/main/test/scheduler/tainttoleration_integration_test.go#L210-L281

Reply:
interesting, we had an add taint test but didn't implement remove taint.